lib/src/blocs/crypto_detail/crypto_detail_bloc.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:bloc/bloc.dart';
+import 'package:equatable/equatable.dart';
+
+part 'crypto_detail_event.dart';
```

Comment:
![image](https://user-images.githubusercontent.com/39506270/140873043-553d55fe-e8a2-4b8c-a2cc-0047916ebec4.png)
from: https://dart.dev/guides/libraries/create-library-packages#organizing-a-library-package

Let's avoid using `part`/`part of` as per recommendations.

Reply:
I fixed it.